inference-gateway/migrations/20240626175841_fdw.sql

```diff
@@ -0,0 +1,29 @@
+CREATE EXTENSION IF NOT EXISTS postgres_fdw;
+CREATE EXTENSION IF NOT EXISTS pg_cron;
+
+CREATE SERVER IF NOT EXISTS cp_queue_server
+FOREIGN DATA WRAPPER postgres_fdw
+OPTIONS (host 'localhost', port '5432', dbname 'postgres');
+
+CREATE USER MAPPING IF NOT EXISTS FOR postgres
+SERVER cp_queue_server
+OPTIONS (user 'postgres', password 'postgres');
+
+CREATE FOREIGN TABLE IF NOT EXISTS fdw_paid_organizations (
+    organization_id text NOT NULL,
+    has_cc boolean not null default false
+)
+SERVER cp_queue_server
+OPTIONS (schema_name 'billing', table_name 'paid_organizations');
```

Comment:
See https://linear.app/tembo/issue/SPAE-255/advanced-request-validation-in-inference-gateway